arch/arm/src/sama5/sam_adc.c

```diff
@@ -1890,7 +1889,7 @@ static void sam_adc_gain(struct sam_adc_s *priv)
   /* Set GAIN0 only.  GAIN0 will be used for all channels. */
 
   sam_adc_putreg(priv, SAM_ADC_CGR, ADC_CGR_GAIN0(CONFIG_SAMA5_ADC_GAIN));
-#endif /* CONFIG_SAMA5_ADC_ANARCH */
+   #endif /* CONFIG_SAMA5_ADC_ANARCH */
```

Comment:
```suggestion
#  endif /* CONFIG_SAMA5_ADC_ANARCH */
```

Reply:
not fix yet.

Reply:
I reverted all my changes that included this fix when I was trying to work out why checkpatch.sh was reporting so many what seemed to be daft errors...forgot to put this one back in as i was concentrating on sam_tc.c

I'll double check I did include all the other suggestions.